EIPS/eip-token-standard.md

```diff
@@ -0,0 +1,129 @@
+This is the suggested template for new EIPs.
+
+Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
+
+## Preamble
+
+    EIP: <to be assigned> (I Suggest 20)
+    Title: <EIP title>
+    Author: Fabian Vogelsteller <fabian@ethereum.org>, Vitalik Buterin <vitalik.buterin@ethereum.org>
+    Type: Informational
+    Category ERC
+    Status: Draft
+    Created: 2015-11-19
+
+
+## Simple Summary
+
+Token standard interface.
+
+
+## Abstract
+
+The following standard allows for people to implement a token standard API withing their smart contracts.
```

Comment:
Would also revise this abstract:

"The following standard allows for the implementation of a standard API for tokens within their smart contracts: it primarily provides basic functionality to transfer tokens and allow them to be approved to be spent by another on-chain third party."